brainmeats/finance.py

```diff
@@ -90,18 +90,20 @@ def btc(self):
         gdax_url = 'https://api.gdax.com/products/BTC-USD/ticker'
         g_request = Browser(gdax_url)
         try:
-            gdax = '$%.2f' % float(g_request.json()['price'])
+            g_json = g_request.json()
+            gdax = '$%.2f' % float(g_json['price'])
+            if self.values:
+                gdax = locale.currency(float(g_json['price']) * float(self.values[0])
```

Comment:
g_json['price'] = price":"2399.10000000" format, so nope, not multiplying on gdax value